api/firmware/attestation_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018-2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firmware_test
```

Comment:
it's in _test so that `&configMock{}, &communicationMock{} &loggerMock{}` are available. Should probably move those to `api/firmware/mock` and get rid of `_test` here.